src/loops/nested_loops/hard/q7/BoxMeNow.java

```diff
@@ -0,0 +1,39 @@
+package loops.nested_loops.hard.q7;
+
+/*
+Write a program called BoxMeNow to ask the use for n, and print the following pattern using nested loops based on their
+input. If the input is less than zero print “Invalid input!”
+Sample Output for n =  4:
+Enter number of lines:
+4
+1111
```

Comment:
I think we should change the pattern to have a space between the ones so it actually will stay a box

Reply:
ie `1 1 1 1`